internal/storage/parser_test.go

```diff
@@ -178,9 +179,7 @@ func TestWrite(t *testing.T) {
 			return
 		}
 
-		if exp, got := `{
-  "name": "Tom"
-}
+		if exp, got := `{"name":"Tom"}
```

Comment:
Could you update this so that the JSON output it pretty-printed with 2 spaces please? This is to maintain backward compatibility with prior versions.

If you need non-pretty-printed output then this can be added in another pull request, preferably activated by a command-line flag.